zerver/lib/actions.py

```diff
@@ -544,6 +544,21 @@ def do_set_realm_waiting_period_threshold(realm, threshold):
     )
     send_event(event, active_user_ids(realm))
 
+def do_set_realm_notifications_stream(realm, stream_name):
+    # type: (Realm, Text) -> None
+
+    stream = get_stream(stream_name, realm)
```

Comment:
this function should be passed a stream from its caller, rather than calling `get_stream`.  Our convention is functions with a `do_foo` name should not be able to fail, and this can fail if the stream name is invalid.  

(The caller should use `access_stream`, not `get_stream`, as well, for proper security checks and error handling)